model_analyzer/config/input/config_command_profile.py

```diff
@@ -581,6 +581,14 @@ def _add_triton_configs(self):
                 description=
                 'A dictionary of name-value labels to set metadata for the Triton '
                 'server docker container in docker launch mode'))
+        self._add_config(
```

Comment:
Have you confirmed that if the user doesn't pass in anything, that the docker still has the default shm?

Reply:
yes. manually